src/main.rs

```diff
@@ -1,3 +1,6 @@
+#[macro_use]
+extern crate lazy_static;
```

Comment:
You don't need `extern crate`, you can do

```rust
use lazy_static::lazy_static;

lazy_static! {
}
```

Reply:
True! Changing it.